tracing-subscriber/src/registry/sharded.rs

```diff
@@ -89,6 +89,17 @@ fn id_to_idx(id: &Id) -> usize {
     id.into_u64() as usize - 1
 }
 
+// CloseGuard is used to track how many Registry-backed Layers have
+// processed an `on_close` event. Once all Layers have processed this
+// event, the registry knows that is able to safely remove the span
+// tracked by `id`.
+//
+// For additional details, see the comment on `Registry::start_close`.
```

Comment:
Nit: even though these are private APIs, I think we should use three-slash doc comments for them...not a blocker.

Reply:
Added.